src/Models/User.ts

```diff
@@ -0,0 +1,25 @@
+import {DataTypes, InitOptions, Model, ModelAttributes, Sequelize} from "sequelize";
+
+export class User extends Model {
+
+    private email: string;
+    private hashPassword: string;
+
+    public static initialize(sequelize: Sequelize) {
+        this.init({
+            email: {
+                type: DataTypes.STRING(128),
+                allowNull: false,
+            },
+            // eslint-disable-next-line
```

Comment:
what's disabled here?

Reply:
Eslint was complaining about hash_password not being in camel case, the column name in the User table is in snake case and because of that it has to be in snake case here.